src/cpp/rtimer.cpp

```diff
@@ -244,4 +246,10 @@ void RTimer::init(int capacity)
 	g_manager = new TimerManager(capacity);
 }
 
+void RTimer::deinit()
+{
+	delete g_manager;
+	g_manager = 0;
```

Comment:
Is there a reason we use 0 instead of NULL or nullptr?

Reply:
For consistency with the rest of the code, which uses 0 for null due to originally using a pre-C++11 dialect.

Would not be opposed to a PR that changes all pointer nullings to nullptr. :)